### DISCUSSION TITLE
use in multi tenant

### DISCUSSION THREAD

**Discussion by vahidalvandi** [spatie/laravel-settings#176] [Q&A]
hi . i want to use this package in multi tenant app . Each tenant has its own settings .
how i can run it in multi tenant

**Comment by rubenvanassche** [comment]
Take a look here: https://github.com/spatie/laravel-settings/discussions/39

This package can be used in multi tenancy setups but it requires some work. We use it with the spatie/multi-tenancy package wothout problems but this requires some integration.

Since we want to keep this package as light as possible we're not providing any bootstrap for external packages since this would only clutter the code in my opinion.
